skycoin/skywire
# Fix IP issue on survey

Did you run `make format && make check`? Yes

Fixes #_

 Changes:	
- remove fetching IP from `ip.skycoin.com`
- add `FetchIP` method to get it by dmsg servers

How to test this PR:
- build
- run `skywire cli survey`
- check generated survey, section `ip_address`. It should just have IP address. Something like this  
  ![image](https://github.com/user-attachments/assets/cb8a2e1f-2483-40de-bf5b-f53f090a4991)
